frontend/src/components/AllQueues.js

```diff
@@ -128,6 +142,17 @@ const AllQueues = observer(() => {
         </Tabs>
       </AppBar>
       <QueueTable queueData={tabValue + 1} />
+
+      <div
+        style={{
```

Comment:
best to avoid inline styles, can you create a scss/queue file and add this there